index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="./style.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Lato:wght@300;400;700;900&family=Montserrat:wght@400;700&family=Playfair+Display:wght@400;500&family=Poppins:wght@100;300;400&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <ul>
+        <li><img src="icons/toolbar/facebook.png" alt="facebook-ribbon-icon"></li>
+        <li><img src="icons/toolbar/twitter-sign.png" alt="twitter-ribbon-icon"></li>
+        <li><select name="language" id="language">
+          <option value="english">English</option>
+          <option value="swahili">Kiswahili</option>
+        </select></li>
+        <li>My Page</li>
+        <li>Login</li>
+      </ul>
+    </div>
+
+    <nav class="navbar">
+      <a href="index.html"><img class="logo" src="icons/MotoImg/Moto_Gp_logo.svg.png" alt="MotoGP"></a>
+      <ul class="nav-menu">
+          <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+          <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+          <li class="nav-item"><a href="#drivers" class="nav-link">Riders and Teams</a></li>
+          <li class="nav-item"><a href="#races.html" class="nav-link">Races</a></li>
+      </ul>
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+
+      <ul class="desktop-nav">
+          <li><a href="about.html" class="page-links-1">About</a></li>
+          <li><a href="#drivers" class="page-links">Our Riders</a></li>
+          <li><a href="#race-days" class="page-links">Races</a></li>
+          <li><a href="#https://www.formula1.com/en/latest.html" class="page-links">News</a></li>
+          <li><a href="https://www.formula1.com/en/latest/tags.race-calendar.2Sa4DEdBSUMcOqa8aYceEe.html" class="page-links campaign" id="campaign">F1 Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+
+    <section id="moto-home">
+      <q class="subtitle">Welcome to the new MotoGP beta website!</q>
+  
+      <h1>Monster Energy British grand Prix</h1>
+      <h2 class="year">2023</h2>
+      <p class="annotation">BritishGP <span>TALKING POINTS</span> MotoGP News ...</p>
+      <q class="news">I've always said Aleix has the biggest balls in the paddock</q>
+      <p class="date">2023.08.06(SUN)</p>
+      <p class="address">@MotoGP, British Grand Prix</p>
+    </section>
```

Comment:
I'd suggest you reduce the width of the image here as it looks too close to the margin and it's different from our deisgn given 😄 

Figma design :
![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/20537712/e5c1ce3b-c6bf-4743-8b86-ce461bd220ab)

Your design :
![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/20537712/c6fb36ba-4070-4bd1-adef-e844e8241db5)